app/views/announcements/_form.html.erb

```diff
@@ -1,20 +1,29 @@
-<%= simple_form_for @announcement , html: { multipart: true}  do |f| %>
+<%= simple_form_for @announcement , html: { multipart: true, class: 'form-horizontal' }  do |f| %>
   <%= f.error_notification %>
 
-  <%= f.label "Starts At:" %>
-  <%= f.text_field :starts_at, value: @announcement.starts_at.strftime('%m/%d/%Y'), class: 'date_start'%>
-  <%= f.input :starts_at, as: :hidden, input_html: {id: 'date_start_alt'} %>
-
-  <%= f.label "Ends At:" %>
-  <%= f.text_field :ends_at, value: @announcement.ends_at.strftime('%m/%d/%Y'), class: 'date_end' %>
-  <%= f.input :ends_at, as: :hidden, input_html: {id: 'date_end_alt'} %>
+  <div class="form-group">
+    <%= f.label :starts_at, 'Starts At:', class: 'col-sm-2 control-label' %>
+    <div class="col-sm-7">
+      <%= f.text_field :starts_at, value: @announcement.starts_at.strftime('%m/%d/%Y'), class: 'date_start form-control' %>
+      <%= f.input :starts_at, as: :hidden, input_html: {id: 'date_start_alt'} %>
+    </div>
+  </div>
+  <div class="form-group">
+    <%= f.label :ends_at, 'Ends At:', class: 'col-sm-2 control-label' %>
+    <div class="col-sm-7">
+      <%= f.text_field :ends_at, value: @announcement.ends_at.strftime('%m/%d/%Y'), class: 'date_end form-control' %>
+      <%= f.input :ends_at, as: :hidden, input_html: {id: 'date_end_alt'} %>
+    </div>
+  </div>
 
   <%#= f.label "Visible To" %>
```

Comment:
We don't have this parameter in the `Announcement` model, let's just delete these lines.